app.js

```diff
@@ -4,13 +4,28 @@ const path = require('path');
 const cookieParser = require('cookie-parser');
 const logger = require('morgan');
 const CORS = require('./src/middlewares/cors');
+const cron = require('node-cron');
+const controller = require('./src/components/upload/controller');
 require('dotenv').config();
 
 const news_api = require('./src/components/news-api/index');
 const indian_stats_api = require('./src/components/indian-stats-api/index');
```

Comment:
you should place `const upload = require('./src/components/upload/index');` here for consistency.